File: server/Models/Post.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Server.Models
{
    public class Post
    {
        [Key]
        public int id { get; set; }

        public string text { get; set; }

        public string thema { get; set; }

        public int userId { get; set; }

        [ForeignKey("userId")]
        public virtual User user { get; set; }

        public virtual List<Comment> comments { get; set; }

    }
}

File: server/app/Models/Post.php
<?php


namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Post extends Model
{

 protected $fillable = [
     'thema',
     'text',
 ];

    public $timestamps = false;


 public function user() {
    return $this->hasOne('App\Models\User');
 }

 public function comments() {
     return $this->hasMany('App\Models\Comment');
 }
}
